src/main/java/org/gridsuite/actions/server/entities/AbstractContingencyEntity.java

```diff
@@ -36,8 +33,6 @@ public abstract class AbstractContingencyEntity {
     @Column(name = "id")
     private UUID id;
 
-    @LastModifiedDate
```

Comment:
why did you remove this annotation?

https://docs.spring.io/spring-data/jpa/reference/auditing.html  
https://docs.spring.io/spring-data/data-commons/docs/current/api/org/springframework/data/annotation/LastModifiedDate.html